app/src/components/features/rules/RulePairs/Pairs/Rows/RowsMarkup/RequestSourceRow/index.js

```diff
@@ -126,23 +127,28 @@ const RequestSourceRow = ({ rowIndex, pair, pairIndex, ruleDetails, isInputDisab
   const renderSourceKeys = useMemo(() => {
     return (
       <Menu>
-        {sourceKeys.map(({ id, title, ruleKey }) => (
-          <Menu.Item
-            key={id}
-            onClick={(event) => {
-              dispatch(
-                actions.updateRulePairAtGivenPath({
-                  pairIndex,
-                  updates: {
-                    [APP_CONSTANTS.PATH_FROM_PAIR.RULE_KEYS]: ruleKey,
-                  },
-                })
-              );
-            }}
-          >
-            {title}
-          </Menu.Item>
-        ))}
+        {sourceKeys.map(({ id, title, ruleKey }) => {
+          if (isExtensionManifestVersion3() && ruleKey === GLOBAL_CONSTANTS.RULE_KEYS.PATH) {
+            return null;
+          }
```

Comment:
This case is pretty rare as we already have migrations. But in case user reaches here, instead of showing null, we should select domain automatically instead of path

Reply:
This hides the path operator from UI

Reply:
Makes sense